### ISSUE TITLE
Use vimium search as early as default browser page search

### ISSUE THREAD

**Issue by kaldown** [philc/vimium#3003]
When bit page opens, vimium won't let you use search `/`, but default browser (^F) works find.
Is it possible to so something with it?
Most of the time I need to look for a man page and insta close it, what for I must wait whole page to load.

**Comment by smblott-github** [comment]
> Is it possible to so something with it?

I don't think so.  We just have to wait until the browser runs our Javascript.
